code/modules/mob/living/logout.dm

```diff
@@ -4,3 +4,6 @@
 	if(!key && mind) //key and mind have become separated.
 		mind.active = FALSE //This is to stop say, a mind.transfer_to call on a corpse causing a ghost to re-enter its body.
 	med_hud_set_status()
+
+	if(fov_handler)
+		qdel(fov_handler)
```

Comment:
```suggestion
	QDEL_NULL(fov_handler)
```

Reply:
fov_handler nulls the reference to itself on Destroy()

Reply:
do you trust it?

Reply:
I guess it's 1 line less